src/NepaliDate.ts

```diff
@@ -624,8 +629,27 @@ class NepaliDate {
     }
 
     /**
-     * Creates a NepaliDate instance by parsing a provided English Date and Time string
-     * with the given format.
+     * Parses a Nepali date-time string in Devanagari form according to the
+     * specified format and returns a `NepaliDate` instance.
+     *
+     * @param dateStringNe - The Nepali Date and time string in Devanagari.
+     * @param format - The format of the provided date-time string.
+     * @example
+     * const dateStringNe = '२०८०/०८/१२ १४-०५-२३.७८९'
+     * const format = 'YYYY/MM/DD HH-mm-ss.SSS'
+     * const nepaliDate = NepaliDate.parseNepaliFormat(dateStringNe, format)
+     */
+    static parseNepaliFormat(dateStringNe: string, format: string): NepaliDate {
+        const [year, month0, day, hour, minute, second, ms] = parseNepaliFormat(
+            dateStringNe,
+            format
+        )
+        return new NepaliDate(year, month0, day, hour, minute, second, ms)
```

Comment:
```suggestion
        return new NepaliDate(...parseNepaliFormat(
            dateStringNe,
            format
        ))
```

makes sense to shorten this as the variables are not being used in any way

Reply:
Got an error with message "A spread argument must either have a tuple type or be passed to a rest parameter.". Leaving it as it is before.